pootle/settings/50-project.conf

```diff
@@ -92,6 +92,7 @@ INSTALLED_APPS = [
     'pootle_profile',
     'pootle_statistics',
     'pootle_terminology',
+    'pootle_fs',
```

Comment:
ive removed it from the default settings for now.

I think as its beta its better to explicitly enable it.